backend/sql/initial_schema.sql

```diff
@@ -0,0 +1,103 @@
+create table project
+(
+    project_id  bigserial not null primary key,
+    name        text      not null unique,
+    description text      null
+);
+
+create table test_methodology
+(
+    test_methodology_id bigserial     not null primary key,
+    name                text          not null unique,
+    description         text          null,
+    confluence_link     varchar(1000) null
+);
+
+-- Corresponds to an EXCEL file
+create table experiment
+(
+    experiment_id  bigserial                not null primary key,
+    project_id     bigint                   not null,
+    methodology_id bigint                   not null,
+    -- This is input from the user
+    name           text                     null,
+    meta           text                     null,
+    -- This is fallback name to show if name is not specified
+    raw_file_name  text                     not null,
+    upload_date    timestamp with time zone not null,
+
+    constraint belongs_to_project
+        foreign key (project_id) references project (project_id)
+            on update cascade
+            on delete cascade,
+
+    constraint has_methodology
+        foreign key (methodology_id) references test_methodology (test_methodology_id)
+            -- Methodology_id is immutable
+            on update no action
+            -- Prevent removing methodology if there is an experiment having such methodology
+            on delete no action
+);
+
+create table measurement
+(
+    measurement_id bigserial        not null primary key,
+    experiment_id  bigint           not null,
+    -- Either compound id or name
+    compound_id    text             not null,
+    concentration  double precision not null,
+    signal         double precision not null,
+
+    constraint belongs_to_experiment
+        foreign key (experiment_id) references experiment (experiment_id)
+            on delete cascade
+            on update cascade
+);
+
+create table analysis_method
+(
+    analysis_method_id bigserial not null primary key,
+    description        text      null,
+    -- Analysis-specific parameters
+    parameters         json      null
+);
+
+create table analysis
```

Comment:
As I said below, it should be an aggregate / meta table, tracking some measurements.
As customers requested, it should have some important metadata: 

 - Confidence of the lab assistant that this data is good vs bad (@gromakovsky that's what I call "high quality data", sets of measurements that were green-lit by a lab assistant)
 - IC50 value that was obtained by this analysis

Also, we'll have important data *derived*:

 - List of other analyses for the same experiment
 - Within how many deviations is the calculated IC50 value from currently established IC50 value for the given compound (see top level review comment)

Reply:
>IC50 value that was obtained by this analysis

it's supposed to be in `result` array, as I am not sure which (and how many) values IC50 produces (I think at least two: IC50 and deviation or something like this)

Reply:
We can have just two numbers there, because that's what we are going to compute in the MVP, or arbitrary JSON, because other analysis methods may produce something more complex and JSON looks like a nice way of abstraction. I am fine with both options.

Reply:
> Confidence of the lab assistant that this data is good vs bad 

AFAIU we agreed to call it `is_suspicious` flag.